src/Symfony/Bridge/PhpUnit/bin/simple-phpunit.php

```diff
@@ -93,7 +93,10 @@
     }
 };
 
-if (\PHP_VERSION_ID >= 80000) {
+if (\PHP_VERSION_ID >= 80100) {
+    // PHP 8.1 requires PHPUnit 9.5+
+    $PHPUNIT_VERSION = $getEnvVar('SYMFONY_PHPUNIT_VERSION', '9.5') ?: '9.5';
+} elseif (\PHP_VERSION_ID >= 80000) {
     // PHP 8 requires PHPUnit 9.3+
     $PHPUNIT_VERSION = $getEnvVar('SYMFONY_PHPUNIT_VERSION', '9.4') ?: '9.4';
```

Comment:
@derrabus maybe this line should be updated to use `9.5` instead. AFAIK, PHPUnit does not ship patch releases for minor versions older than the last one of each major.

Reply:
With that argument, we should as well bump from 8.3 to 8.5 for PHP 7.2.

@nicolas-grekas Can you advise on our policy regarding the defaults in that file?

Reply:
works for me, let's give it a try

Reply:
@derrabus makes sense. To me, we should provide maintained minor versions of PHPUnit for the default values (for the major versions that are still maintained)

Reply:
Done!